blocks/simple-list-block/package.json

```diff
@@ -25,11 +24,7 @@
     "@arc-fusion/prop-types": "^0.1.5",
     "@wpmedia/engine-theme-sdk": "*",
     "@wpmedia/news-theme-css": "*",
```

Comment:
this `news-theme-css` dependency can be removed as it's not used any more

Reply:
```suggestion
```

Reply:
Fixed

Reply:
cool thanks -- sorry for nitpick